src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -248,7 +248,7 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
                 SimpleBlobStoreFactory.create(
                     remoteOptions,
                     GoogleAuthUtils.newCredentials(authAndTlsOptions),
-                    env.getWorkingDirectory()),
+                    Preconditions.checkNotNull(env.getWorkingDirectory())),
```

Comment:
nit: `Preconditions.checkNotNull(env.getWorkingDirectory(), "workingDirectory")` having some string associated with this check is useful when receiving crash logs.